docs_CN/RxSwift.md

```diff
@@ -1,4 +1,4 @@
-# RxSwift
+# RxSwift（参照ReactiveSwift）
```

Comment:
This is RxSwift, not ReactiveSwift. Why did you add this?

Reply:
because description is the same。so Translation is the same

Reply:
@lovesomeone The descriptions may be similar but the syntax provided in the code snippets is different. I think that there should be a `RxSwift.md` & `ReactiveSwift.md` just like the English docs.

Reply:
Ok  you're right . It may be ambiguous.   I'll update it 。Keep them  separate。